src/components/playback/mediasession.js

```diff
@@ -97,6 +97,15 @@ define(['playbackManager', 'nowPlayingHelper', 'events', 'connectionManager'], f
     }
 
     function updatePlayerState(player, state, eventName) {
+        var now = new Date().getTime();
+        lastUpdateTime = now;
+
+        // Don't go crazy reporting position changes
+        if (eventName == 'timeupdate' && (now - lastUpdateTime) < 5000) {
```

Comment:
lastUpdateTime = now;
now - lastUpdateTime = 0

Reply:
Wow, completely missed that. It's crazy to see what we got from Emby codebase!

Thanks for pointing it out!